core/templates/dev/head/css/oppia.css

```diff
@@ -3917,15 +3917,16 @@ md-card.preview-conversation-skin-supplemental-card {
 }
 
 .oppia-about-three-cols-layout {
+  display: inline-block;
   margin-right: 20px;
-  width: calc((100% - 40px) / 3);
+  width: calc((100% - 60px) / 3);
```

Comment:
Might be best to provide a default width in case there are those with legacy browsers.